platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/server/IdpEndpoint.java

```diff
@@ -1383,6 +1386,8 @@ private Response continueLogout(
           return continueLogout(logoutState, cookie, incomingBinding);
         }
 
+        Instant notOnOrAfter = Instant.now().plus(30, ChronoUnit.MINUTES);
```

Comment:
❓ Is there a reason you chose 30 minutes here?

Reply:
I got that from @stustison. He suggested to try and use the configuration for SAML assertions and if I wasn't able to get that he suggested using 30 minutes.

Reply:
✏️ Should set 30 as a const instead of leaving inline as a magic number.